Conceptpower+Spring/src/main/java/edu/asu/conceptpower/rest/ConceptIDLookup.java

```diff
@@ -88,7 +88,7 @@
 
         XMLConceptMessage msg = messageFactory.createXMLConceptMessage();
 
-        if (entry != null) {
+        if (entry != null && !entry.isDeleted()) {
```

Comment:
the id lookup should still work for deleted concepts, but when searching it shouldn't.